Lib/pathlib/_abc.py

```diff
@@ -206,8 +206,8 @@ class PurePathBase:
     )
     pathmod = os.path
 
-    def __init__(self, *paths):
-        self._raw_paths = paths
+    def __init__(self, *args):
+        self._raw_paths = [arg for arg in args if arg]
```

Comment:
It was a tuple, and now it is a list. Also, does not this processing add an overhead? In common case there should not be empty components (except maybe `Path('')`).

Reply:
It add a tiny bit of overhead in the common case. Passing an empty string does happen, e.g. importlib calls `joinpath('')`, but perhaps not commonly enough to make this worthwhile. I'll undo.